data-archive/archive-jobs/mosip_credential/mosip_archive_credential_table1.py

```diff
@@ -0,0 +1,139 @@
+#-- -------------------------------------------------------------------------------------------------
```

Comment:
Please rename the filename as mosip_archive_credential_table_base-on-statuscode.py

Reply:
done